_posts/2017-10-23-storm-v1.11.13-released.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: StoRM 1.11.13 released
+author: enrico
+summary: Released packages:<br/><span>StoRM Backend 1.11.13</span>
+---
+
+The StoRM Product Team is pleased to announce the release of
+[StoRM 1.11.13][release-notes] that includes the following updated components:
+
+* StoRM Backend v. [1.11.13][backend-rn]
+* YAIM StoRM v. [4.3.10][yaim-rn]
+
+This release provides fixes to some outstanding bugs:
+
+* fixes gemss failure of recall task final status update;
```

Comment:
fixes a problem that prevented correct status update for tape recalls handled by GEMSS